src/pages/index.svelte

```diff
@@ -77,6 +90,21 @@
 		</div>
 	</Card>
 
+	<Dialog
+		bind:this={dialog}
+		aria-labelledby="dialog-title"
+		aria-describedby="dialog-content">
+		<Title id="dialog-title">Message to {$currPlayer.name}</Title>
```

Comment:
I thought adding the player's name was a nice personal touch.

Reply:
yeah, I like that too